components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -140,10 +140,15 @@
           else
             "nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04",
         local installTfma =
-          if tf_version < "1.9" then
+          if tf_version < "1.6" then
             "no"
           else
             "yes",
+        local tfmaVersion =
```

Comment:
Can we get rid of this? and "INSTALL_TFMA"? If TFMA_VERSION is specified then that should mean we should install TFMA and if it isn't specified then we shouldn't install TFMA.